src/Microsoft.VisualStudio.Editor.Razor/RazorDirectiveCompletionProvider.cs

```diff
@@ -205,7 +205,7 @@ protected virtual bool TryGetRazorSnapshotPoint(CompletionContext context, out S
                 }
 
                 var mappedPoints = projectionSnapshot.MapToSourceSnapshots(context.CompletionListSpan.Start);
-                var htmlSnapshotPoints = mappedPoints.Where(p => p.Snapshot.TextBuffer.ContentType.IsOfType(RazorLanguage.ContentType));
```

Comment:
nit: this doesn't really need to change. We'll never really operate on non-core, but this is still true for both content types.

Reply:
Should just be using the `IsRazorBuffer()` here.